packages/ui/src/molecules/Breadcrumb/Breadcrumb.tsx

```diff
@@ -43,23 +35,22 @@ const ListItem: FC<ListItemProps> = ({ children, isLastItem, testId }) => {
 
   if (!React.isValidElement(children)) {
     return (
-      <li>
+      <li data-breadcrumb-list-item data-breadcrumb-divider={divider}>
```

Comment:
By doing this you can't pass React components to be used as dividers. Only strings aren't sufficient for our use case.

Reply:
Do you have any other alternative to fix the HTML validation? I can only think of moving the divider inside the `li`, as a sibling to `ListItem`'s children? Or maybe we should just ignore this validation?

Reply:
Hey @igorbrasileiro! 

I ultimately went with the approach of placing the divider inside the `li` instead. Maybe it's easier to understand with this comparison:

Before|After
-|-
<img width="416" alt="HTML" src="https://user-images.githubusercontent.com/381395/155134958-d3f202c0-b199-4e5f-ac67-df1b96b560c2.png">|<img width="416" alt="HTML" src="https://user-images.githubusercontent.com/381395/155134993-695e8284-461e-4731-b8f9-ae741804e794.png">

I had initially liked [your suggestion](https://github.com/vtex/faststore/pull/1143#discussion_r803874204), but after some thought, I conclude it would clutter the number of `li`s, with a number greater than it's the passed `children`. This new way also simulates a little bit where the [CSS `before` pseudo-element would show up in the DOM](https://user-images.githubusercontent.com/381395/153265283-bbd114ce-84af-4b56-8311-bf2cc92b75b5.png) if we had gone with that approach.

This final approach also keeps the use case of letting users use any HTML element as the divider, not just strings.

Since I had to change the implementation, could you take a new look? No rush though!

Reply:
Yeah, you had a better solution!! Great job